ydb/core/tx/schemeshard/ut_base/ut_base.cpp

```diff
@@ -10079,7 +10079,7 @@ Y_UNIT_TEST_SUITE(TSchemeShardTest) {
 
     Y_UNIT_TEST(RejectSystemViewPath) {
         TTestBasicRuntime runtime;
-        TTestEnv env(runtime, 4, true, &CreateFlatTxSchemeShard, true);
+        TTestEnv env(runtime);
```

Comment:
This test should explicitly set EnableSystemViews to true via `TTestEnvOptions().EnableSystemViews(true)` as the test is meaningless if `EnableSystemViews = false`.
And it does not matter that `EnableSystemViews = true` by default -- default could be changed (not in practice but in theory).